static/js/muting_ui.js

```diff
@@ -158,6 +163,59 @@ export function toggle_topic_mute(message) {
     }
 }
 
+export function persist_user_mute(user_id) {
+    last_user_update = timestamp_ms();
+    channel.post({
+        url: "/json/users/me/muted_users/" + user_id,
+        idempotent: true,
+    });
+}
+
+export function persist_user_unmute(user_id) {
+    last_user_update = timestamp_ms();
+    channel.del({
+        url: "/json/users/me/muted_users/" + user_id,
+        idempotent: true,
+    });
+}
+
 export function handle_user_updates(muted_user_ids) {
+    popovers.hide_user_info_popover();
+    popovers.hide_message_info_popover();
+    popovers.hide_user_sidebar_popover();
+    if (timestamp_ms() < last_user_update + 1000) {
```

Comment:
This 1000 hack copied from topic-muting seems like a bug there as well.  I don't want to particularly block this on fixing that, but if you have time, I don't think the Zulip API ever has a use case for 1s timeouts like this, and we should figure out what other bug the topic_mute code there is masking and fix that (and then remove it).

Reply:
I opened #18279 for this.